cranelift-codegen/src/isa/x86/binemit.rs

```diff
@@ -341,3 +341,20 @@ fn jt_disp4<CS: CodeSink + ?Sized>(jt: JumpTable, func: &Function, sink: &mut CS
     sink.put4(delta);
     sink.reloc_jt(Reloc::X86PCRelRodata4, jt);
 }
+
+/// Emit a four-byte displacement to `constant`
+fn const_disp4<CS: CodeSink + ?Sized>(
+    constant: impl Into<Constant>,
+    _func: &Function,
+    sink: &mut CS,
+) {
+    //let delta = func.constant_offsets[constant].wrapping_sub(sink.offset() + 4);
+    sink.put4(0); // TODO emit four zeroes
```

Comment:
@sunfishcode mentioned that I should be emitting the relative offset of the constant as zeroes here and expect the embedder to fix this once they are informed by `RelocSink::reloc_constant`. However, for the `MemorySink` I do not see where this is done (or if it is done) and from looking at the jump table implementation I suspect that for `MemorySink` to work we need to put a delta here. Is this an issue with the jump table or am I understanding the jump table code (see `jt_disp4` above) incorrectly?